input/submitters/dcc52.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id" : "dcc52",
+  "name" : "Daniel Carter",
+  "email" : "dcc52@cam.ac.uk",
+  "url" : "",
```

Comment:
Do you want to put up a web page somewhere if you haven't already?

Reply:
You're right, that's probably worth doing and I'll put something up (possibly at the SRCF unless you can suggest somewhere better?)